src/cf.data.pre

```diff
@@ -6454,9 +6454,16 @@ DOC_START
 	to be erroneously cached unless the application designer
 	has taken the appropriate actions.
 
-	'Percent' is a percentage of the objects age (time since last
-	modification age) an object without explicit expiry time
-	will be considered fresh.
+	'Percent' is used to compute the expiration time of responses with a
+	Last-Modified header and without an explicit expiry time: When Squid
+	caches (or updates) such a response, the expiration time is effectively
+	set to the given percentage of the time passed since last modification.
+	For example, if such an object was cached 1000 minutes after its
+	Last-Modified time and percent is set to 20%, then the object will be
+	considered fresh for the first 200 minutes (1000*0.20=200) it spends in
+	the cache. The lm-factor term used in the algorithm sketch below is object
+	age divided by stable-time, where stable-time is the time between the last
+	object modification and the moment the object was cached (or updated).
```

Comment:
IMO, too many readers will think that `Date` in the above suggestion is the current transaction time rather than the Date header of the previously cached response.

I am also concerned that folks will misinterpret this as Squid actually adding the `Cache-control: max-age` header to the response.

Finally, the above version does not fix the `lm-factor` problem.

Nevertheless, I have merged this suggestion and adjusted PR description accordingly (please adjust further as needed). When you are happy with the result, please clear this PR for merging.